src/System.Private.CoreLib/shared/System/Buffer.cs

```diff
@@ -112,34 +112,6 @@ public static unsafe void MemoryCopy(void* source, void* destination, ulong dest
             Memmove((byte*)destination, (byte*)source, checked((nuint)sourceBytesToCopy));
         }
 
-        internal static unsafe void Memcpy(byte[] dest, int destIndex, byte* src, int srcIndex, int len)
```

Comment:
VM depends on these methods as well here: https://github.com/dotnet/coreclr/blob/master/src/vm/mscorlib.h#L937

Reply:
yeah (moved it to .CoreCLR.cs version)